runtime/tutor.txt

```diff
@@ -416,6 +415,338 @@ _________________________________________________________________
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Select each wrongly capitalised or lowercase letter
+    and press ~ over them.
+ (remember, the first letter *should* be capitalised)
+
+ --> thIs sENtencE hAs MIS-cApitalIsed leTTerS.
+
+
+
+
+
+
+
+
+
+
+=================================================================
+=                         SETTING CASE                          =
+=================================================================
+
+ Press ` (above tab key on most keyboards) to set all selected
+ letters to lowercase.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Press x to select the line.
+ 3. Press ` to change it to all lowercase.
+
+--> this SENTENCE SHOULD all be in LOWERCASE!
+
+ Note: Conversely, alt-` changes all selected letters to uppercase.
+
+
+
+
+
+
+
+
+=================================================================
+=                            MACROS                             =
+=================================================================
+
+ Macros are a way to record a set of actions that you want
+ to repeat.
+
+ Press Q to start recording a macro, you should see a popup at
+ the bottom of your screen. Press Q again to stop recording.
+ Press q to repeat your recorded macro.
+
+ 1. Move the cursor to the first line below marked -->.
+    Ensure your cursor is on the > of the arrow.
+ 2. Press Q to start recording.
+ 3. Edit the line to look like the bottom one.
+ 4. Exit insert and press Q again to stop recording.
+ 5. Move to the line below and put your cursor on the > again.
+ 6. Press q to repeat the macro.
+
+ --> ... sentence doesn't have it's first and last ... .
+ --> ... sentence doesn't have it's first and last ... .
+ --> This sentence doesn't have it's first and last word.
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Press y to yank (copy) text and p to paste.
+   * Press space-Y and space-P to yank/paste on the system
+     clipboard.
+
+ * Press ~ to alternate case of selected letters.
+   * Press ` and alt-` to set the case of selected layers to
+     upper and lower respectively.
+
+ * Press Q to record and stop recording a macro.
+   * Press q to repeat the recorded macro.
+
+
+
+
+
+
+
+
+=================================================================
+=                      USING THE JUMPLIST                       =
+=================================================================
+
+ Helix can keep track of "jumps" which are big movements, like
+ jumping to the definition of a function in code. It stores
+ these in what's called the jumplist.
+
+ Press C-s (ctrl-s) to manually save your current position to
+ the jumplist.
+
+ Press C-i and C-o to move forward and backwards in the jumplist
+ respectively.
+
+ 1. Press C-s somewhere.
+ 2. Move far away in the file.
```

Comment:
Not quite true, not all movements are being recorded to jump list, but IIRC `ctrl-u` can save the jump list manually, I don't find that useful though.

Reply:
I say "big movements, like jumping to a definition in code." Is that not true? C-u is Move half page up and I agree it's not as central a concept, but still deserves a place somewhere.

Reply:
I'm still not sure what you want changed specifically, does the introduction sentence have something incorrect?

Reply:
Maybe that is better yes.